docs/api-spec.md

```diff
@@ -351,6 +351,55 @@ token needs to be presented when creating a replica of this primary account.
 Sublease tokens can only be issued for primary accounts. If the account in question is a replica account, 400 (Bad
 Request) is returned.
 
+## GET /keppel/v1/accounts/:name/trivy\_policies
+
+If this Keppel is configured to use its bundled [Trivy security scanner](https://aquasecurity.github.io/trivy), this
+endpoint returns a list of user-defined policies for how Keppel processes reports generated by Trivy.
+On success, returns 200 and a JSON response body like this:
+
+```json
+{
+  "policies": [
+    {
+      "match_repository": ".*",
+      "only_unfixed": true,
+      "action": {
+        "ignore": true,
+        "assessment": "risk accepted: vulnerabilities without an available fix are not actionable"
+      }
+    },
+    {
+      "match_repository": "my-python-app|my-other-image",
+      "match_vulnerability": "CVE-2022-40897",
```

Comment:
Maybe we should add a match severity if it would be a use case to ignore low, unfixed CVEs.

Reply:
This can be added later if the need arises. For now, I'll confine myself to the feature set that we discussed with the Heureka team.